src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
I would hold off on the change on removing net46. As pointed out by @natemcmaster, it seems like Razor still cross-compiles net46 in its packages and tests (https://github.com/aspnet/Razor/search?utf8=%E2%9C%93&q=net46&type=). Removing net46 will probably break it. Let's go with the extra `#if`s for now.